src/modules/AddressBook.js

```diff
@@ -0,0 +1,238 @@
+'use strict';
+
+var Utils = require('../qbUtils');
+var config = require('../qbConfig');
+
+/**
+ * Address Book
+ * @namespace QB.addressbook
+ */
+function AddressBook(service) {
+  this.service = service;
+}
+
+AddressBook.prototype = {
+  /**
+   * Method cud is create, update and delete contacts in address book. <br />
+   * An operation choosens by state: if contacts (to identify uses phone number) doesn't exist in address book it will save,
```

Comment:
"An operation choosens by state:" - can remove it.

we can simply write the following:

"If contacts doesn't exist in address book then it will be created. If contacts exists then it will be updated. If pass 'destroy: 1' then the contact will be removed. "